apps/predbat/predbat.py

```diff
@@ -204,20 +306,9 @@
         "step": 2,
         "unit": "fraction",
         "icon": "mdi:currency-usd",
-        'default' : 0.0,
-    },
-    {
-        "name": "set_reserve_min", 
-        "friendly_name": 
-        "Set Reserve Min", 
-        "type": "input_number", 
-        "min": 4, 
-        "max": 100, 
-        "step": 1, 
-        "unit": "%", 
-        "icon": "mdi:percent", 
-        'default' : 4.0
+        "default": 0.0,
```

Comment:
Was the change here to 0 a typo?

Reply:
That's Git or GitHub doing a bad matching of lines removed / added, and giving the impression the number changed when it hasn't really. Line 309 in green should be matched with 207 in red. And line 311 in green replaces 209-220 in red.

Reply:
Just to add, the reformatting in here was all automated, so it should be correct, but I'm glad you asked - it worried me when I saw your comment too @springfall2008

Reply:
PS the only reformatting that has been manual was me shortening some of the lines in the Markdown files by breaking long lines into more shorter lines.